Sources/Commands/SwiftTestTool.swift

```diff
@@ -471,16 +484,19 @@ final class TestRunner {
     init(path: AbsolutePath,
          xctestArg: String? = nil,
          processSet: ProcessSet,
-         sanitizers: EnabledSanitizers,
          toolchain: UserToolchain,
-         diagnostics: DiagnosticsEngine
+         diagnostics: DiagnosticsEngine,
+         options: ToolOptions,
+         buildParameters: BuildParameters
     ) {
         self.path = path
         self.xctestArg = xctestArg
         self.processSet = processSet
-        self.sanitizers = sanitizers
         self.toolchain = toolchain
         self.diagnostics = diagnostics
+        self.options = options
+        self.buildParameters = buildParameters
+        self.sanitizers = options.sanitizers
```

Comment:
We can just remove this property all together.

Reply:
done